src/internal_rules.jl

```diff
@@ -672,6 +664,30 @@ function EnzymeRules.forward(::Const{typeof(cholesky)}, RT::Type, A; kwargs...)
     end
 end
 
+function _cholesky_forward(C::Cholesky, Σdot)
```

Comment:
I recommend keeping notation the same as elsewhere
```suggestion
function _cholesky_forward(C::Cholesky, Ȧ)
```

Reply:
I don't think this method is correct for real `Symmetric` or `Hermitian` `A`, since it assumes that `Ȧ` represents the Hermitian tangent of the Hermitian matrix, but in such cases, `Ȧ` is just a container for the tangent of `A.data`. More generally, this rule can be wrong for many structured types that represent Hermitian matrices. I recommend restricting the type of `A` in `forward` and making sure this is thoroughly tested.

Reply:
You may be right, I need to revisit that after https://julialang.slack.com/archives/C01J3R4FHB3/p1709555032435419.

Reply:
Do you have a test that shows that this is wrong?